beta/azure-container-apps/ReadMe.md

```diff
@@ -0,0 +1,228 @@
+    14. Select **Create**.
+    15. Wait for the notification that the revision deployment was successful.
+    16. Select **Overview**.
+    17. On the right hand side, click on the Application Url (this is your SCIM bridge URL), to access and test your SCIM bridge.
+    18. You should be prompted to log into the SCIM bridge with your `bearer token`.
+
+### Step 3: Follow the steps to connect your Identity provider to the SCIM bridge.
+ - [Connect your Identity Provider](https://support.1password.com/scim/#step-3-connect-your-identity-provider)
+
+</details>
+<hr>
+
+## Appendix
+
+### Troubleshooting
+
+Logs for a container app can be viewed from the **Log Stream**, there is a separate log stream for both containers of the revision that is active. Often reviewing the logs of the **op-scim-bridge** container can help understand any startup issues.
+
+### Update 1Password SCIM Bridge
+
+The latest version of 1Password SCIM Bridge is posted on our [Release Notes](https://app-updates.agilebits.com/product_history/SCIM) website, where you can find details about the latest changes. 
+
+1. Start the Azure Cloud Shell from the navigation bar of your [Azure Portal](https://portal.azure.com).
+2. Run the following command, replacing $ContainerAppName ResourceGroup with the names from your deployment. Also ensure to change the version number 2.8.2 to match the latest version from our [SCIM Bridge Release Notes page](https://app-updates.agilebits.com/product_history/SCIM).
+```az containerapp update -n $ContainerAppName -g $ResourceGroup --container-name op-scim-bridge --image docker.io/1password/scim:v2.8.2```
+3. Log into your SCIM bridge URL with your bearer token to validate in the top left hand side that you are running the version of the SCIM Bridge. (logging in with the bearer token will also update your Automated User Provisioning page with the latest access time and with the current version).
+
+<details>
+<summary> Manual Azure Portal Upgrade Steps</summary>
+
+1. Within Container App from the [Azure Container Apps Portal](https://portal.azure.com/#view/HubsExtension/BrowseResource/resourceType/Microsoft.App%2FcontainerApps), Select **Containers** from the left hand side.
+2. Select **Edit and deploy** along the top.
+3. Put a check in the box next to your **op-scim-bridge** container and select **Edit**.
+4. Change the version number **2.8.2** in the **Image and Tag** field, **1password/scim:v2.8.2** to match the latest version from our [SCIM Bridge Release Notes page](https://app-updates.agilebits.com/product_history/SCIM).
+5. Select **Save**.
+6. Select **Create** to deploy a new revision using the updating image.
+7. Log into your SCIM bridge URL with your bearer token to validate in the top left hand side that you are running the version of the SCIM Bridge. (logging in with the bearer token will also update your Automated User Provisioning page with the latest access time and with the current version).
+</details>
```

Comment:
```suggestion
### Update 1Password SCIM Bridge

#### Update using the `az` CLI
The latest version of 1Password SCIM Bridge is posted on our [Release Notes](https://app-updates.agilebits.com/product_history/SCIM) website, where you can find details about the latest changes. 

1. Start the Azure Cloud Shell from the navigation bar of your [Azure Portal](https://portal.azure.com).
2. Run the following command, replacing $ContainerAppName ResourceGroup with the names from your deployment. Also ensure to change the version number 2.8.2 to match the latest version from our [SCIM Bridge Release Notes page](https://app-updates.agilebits.com/product_history/SCIM).
```az containerapp update -n $ContainerAppName -g $ResourceGroup --container-name op-scim-bridge --image docker.io/1password/scim:v2.8.2```
3. Log into your SCIM bridge URL with your bearer token to validate in the top left hand side that you are running the version of the SCIM Bridge. (logging in with the bearer token will also update your Automated User Provisioning page with the latest access time and with the current version).

#### Update using the Azure Portal

1. Within Container App from the [Azure Container Apps Portal](https://portal.azure.com/#view/HubsExtension/BrowseResource/resourceType/Microsoft.App%2FcontainerApps), Select **Containers** from the left hand side.
2. Select **Edit and deploy** along the top.
3. Put a check in the box next to your **op-scim-bridge** container and select **Edit**.
4. Change the version number **2.8.2** in the **Image and Tag** field, **1password/scim:v2.8.2** to match the latest version from our [SCIM Bridge Release Notes page](https://app-updates.agilebits.com/product_history/SCIM).
5. Select **Save**.
6. Select **Create** to deploy a new revision using the updating image.
7. Log into your SCIM bridge URL with your bearer token to validate in the top left hand side that you are running the version of the SCIM Bridge. (logging in with the bearer token will also update your Automated User Provisioning page with the latest access time and with the current version).
```
In line with my thinking above about emphasizing the portal/GUI method, this suggestion puts both cli and portal methods at the same level and makes the language slightly more consistent between the two sections. 


### Update 1Password SCIM Bridge

#### Update using the `az` CLI
The latest version of 1Password SCIM Bridge is posted on our [Release Notes](https://app-updates.agilebits.com/product_history/SCIM) website, where you can find details about the latest changes. 

1. Start the Azure Cloud Shell from the navigation bar of your [Azure Portal](https://portal.azure.com).
2. Run the following command, replacing $ContainerAppName ResourceGroup with the names from your deployment. Also ensure to change the version number 2.8.2 to match the latest version from our [SCIM Bridge Release Notes page](https://app-updates.agilebits.com/product_history/SCIM).
```az containerapp update -n $ContainerAppName -g $ResourceGroup --container-name op-scim-bridge --image docker.io/1password/scim:v2.8.2```
3. Log into your SCIM bridge URL with your bearer token to validate in the top left hand side that you are running the version of the SCIM Bridge. (logging in with the bearer token will also update your Automated User Provisioning page with the latest access time and with the current version).

#### Update using the Azure Portal

1. Within Container App from the [Azure Container Apps Portal](https://portal.azure.com/#view/HubsExtension/BrowseResource/resourceType/Microsoft.App%2FcontainerApps), Select **Containers** from the left hand side.
2. Select **Edit and deploy** along the top.
3. Put a check in the box next to your **op-scim-bridge** container and select **Edit**.
4. Change the version number **2.8.2** in the **Image and Tag** field, **1password/scim:v2.8.2** to match the latest version from our [SCIM Bridge Release Notes page](https://app-updates.agilebits.com/product_history/SCIM).
5. Select **Save**.
6. Select **Create** to deploy a new revision using the updating image.
7. Log into your SCIM bridge URL with your bearer token to validate in the top left hand side that you are running the version of the SCIM Bridge. (logging in with the bearer token will also update your Automated User Provisioning page with the latest access time and with the current version).